docker/build-tools/Dockerfile

```diff
@@ -14,16 +14,11 @@ ENV OUTDIR=/out
 
 # Update distro and install dependencies
 # TODO we probably don't need all this stuff...
+# hadolint ignore=DL3008
 RUN apt-get update && apt-get install -y --no-install-recommends \
-    autoconf \
-    build-essential \
-    libc-ares-dev \
-    libssl-dev \
-    libtool \
     unzip \
     upx \
-    zlibc \
-    zlib1g-dev
+ && rm -rf /var/lib/apt/lists/*
```

Comment:
for future reference, it is not necessary to chain operations together with `&&` in build contexts.  The resulting image size is not smaller (although the activity of deleting /var/lib/apt/lists can be - but not, in this case, as only the toolchain is copied from this context.
t